SonarQube.Client/Api/DefaultConfiguration.cs

```diff
@@ -50,7 +50,7 @@ public static RequestFactory Configure(RequestFactory requestFactory)
                 .RegisterRequest<IGetRoslynExportProfileRequest, V6_60.GetRoslynExportProfileRequest>("6.6")
                 .RegisterRequest<IGetOrganizationsRequest, V7_00.GetOrganizationsRequest>("7.0")
                 .RegisterRequest<IGetIssuesRequest, V7_20.GetIssuesRequestWrapper>("7.2")
-                ;
+                .RegisterRequest<IGetHotspotRequest, V8_1.GetHotspotRequest>("8.1");
```

Comment:
What should we do for older SQ versions? Perhaps we should have some kind of message in the hotspots window. WDYT?

Reply:
(discussed offline, note added to the project board)